### PR TITLE
Feature/delete ideas

### PR DESCRIPTION
**Pull request by rachel-bock**
What does this PR do?
- Add favorite and unfavorite functionality
- Correct general spacing inconsistencies

Where should your reviewer start?
- Line 14 of main.js

What (if any) features are you implementing?
- Being able to mark an idea as a favorite

What (if anything) did you refactor?
- displayCard
- getEventId

Were there any issues that arose?
- No.

Is there anything that you need from your teammate?
- 😁

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

**Comment by BlueNealis** [comment]
Wow oh wow, this code is hotter than a hot dog 🌭🌈.

File: main.js [hunk 1]
```diff
@@ -25,21 +44,33 @@ function processInput() {
 }
 
 //DOM FUNCTIONS
+function updateStarredCard(target) {
+  target.classList.toggle("favorite-active");
+}
+
 function checkInput(){
-  if (inputTitle.value === ""  || inputBody.value === "") {
+  if (inputTitle.value === "" || inputBody.value === "") {
```
Comment (irmakerdem) [inline main.js:52]: Nice getting rid of extra spaces! 🛸 👩‍🚀

File: main.js [hunk 2]
```diff
@@ -25,21 +44,33 @@ function processInput() {
...
 function displayCard() {
   cardGrid.innerHTML = "";
   for (var i = 0; i < list.length; i++) {
     cardGrid.innerHTML += `<div id="${i}" class="card">
       <div class="card-top">
-        <button class="favorite"></button>
-        <button class="delete"></button>
+        <button ${determineClass(list[i])} name="favorite""></button>
+        <button class="delete" name="delete"></button>
       </div>
```
Comment (BlueNealis) [inline main.js:74]: Look at this fancy little number here oh wowo 🥺

File: main.js [hunk 3]
```diff
@@ -25,21 +44,33 @@ function processInput() {
...
+}
+
 function displayCard() {
   cardGrid.innerHTML = "";
   for (var i = 0; i < list.length; i++) {
     cardGrid.innerHTML += `<div id="${i}" class="card">
       <div class="card-top">
-        <button class="favorite"></button>
-        <button class="delete"></button>
+        <button ${determineClass(list[i])} name="favorite""></button>
```
Comment (irmakerdem) [inline main.js:72]: Calling a function in innerHTML!! SICKKK 🤩

**Review by BlueNealis — APPROVED** [review]

**Review by irmakerdem — COMMENTED** [review]
Nice job finishing iteration 3! 🦙

**Comment by rachel-bock** [comment]
Thank you for the feedback!

**Pull request merged**